### PR TITLE
OS X Framework Target

### PR DESCRIPTION
**Pull request by toadzky**

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/SLF4Swift.iml" filepath="$PROJECT_DIR$/.idea/SLF4Swift.iml" />
+    </modules>
+  </component>
+</project>
```
Comment (phimage) [inline .idea/modules.xml]: IntelliJ IDEA file? useful?

**Comment by phimage** [comment]
thanks

File: .idea/modules.xml (hunk 1 above)
Comment (toadzky) [inline .idea/modules.xml]: AppCode actually, but no, not useful. It's a local project file I didn't mean to commit.

**Comment by phimage** [comment]
thanks for carthage

**Comment by toadzky** [comment]
My pleasure.

**Pull request merged**